sdk/storage/storage-file-datalake/test/perfstress/track-2/storageTest.spec.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getEnvVar, PerfStressTest } from "@azure/test-utils-perfstress";
+
+import {
+  DataLakeServiceClient,
+  StorageSharedKeyCredential,
+  DataLakeFileSystemClient,
+  DataLakeDirectoryClient
+} from "../../../src";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+import { generateUuid } from "@azure/core-http";
+dotenv.config();
+
+export abstract class StorageDFSTest<TOptions> extends PerfStressTest<TOptions> {
+  datalakeServiceClient: DataLakeServiceClient;
+  fileSystemClient: DataLakeFileSystemClient;
+  directoryClient: DataLakeDirectoryClient;
+  static fileSystemName = generateUuid();
+  static directoryName = generateUuid();
+
+  constructor() {
+    super();
+    const connectionString = getEnvVar("STORAGE_CONNECTION_STRING");
+    const accountName = getValueInConnString(connectionString, "AccountName");
```

Comment:
Hmm, we should have a connect string constructor for DataLakeServiceClient like in storage-blob.
https://github.com/Azure/azure-sdk-for-js/blob/9e8763b7a333c127bdb9a2629a8720e408bf9f94/sdk/storage/storage-blob/src/BlobServiceClient.ts#L505

Logged https://github.com/Azure/azure-sdk-for-js/issues/13315.

Reply:
Thanks!